packages/create-svelte/shared/+typescript/svelte.config.cjs

```diff
@@ -0,0 +1,26 @@
+const sveltePreprocess = require('svelte-preprocess');
+const node = require('@sveltejs/adapter-node');
+const pkg = require('./package.json');
+
+/** @type {import('@sveltejs/kit').Config} */
+module.exports = {
+	kit: {
+		// By default, `npm run build` will create a standard Node app.
+		// You can create optimized builds for different platforms by
+		// specifying a different adapter
+		adapter: node(),
+
+		// Consult https://github.com/sveltejs/svelte-preprocess
+		// for more information about preprocessors
+		preprocess: sveltePreprocess(),
```

Comment:
Shouldn't this be at the same level as `kit`?

Reply:
good catch, thanks — fixed in 3c41d0756678efe96c887df80205a00816845a0e